docs/mpi-design.md

```diff
@@ -25,21 +25,22 @@ The following diagram illustrates the relationships between the `Person`, `Patie
 ```mermaid
 erDiagram
     Person {
-        bigint id PK "Primary Key"
-        uuid internal_id "Internal UUID"
+        bigint id PK "Primary Key (auto-generated)"
+        uuid reference_id "Reference UUID (auto-generated)"
```

Comment:
What is the purpose of `reference_id` for Patient and Person tables again?

Reply:
Great question, why does a table need two keys!  Really it doesn't, and it could be an over optimization on my part, but here's why.

- **uuids** are great, especially for conveying information back to users as they don't give away any details about the system (auto increments can give attackers information on how big your database is)
- **ints** perform better with joins and take up significantly less database space.

I opted to use the int as the primary key to optimizing joining the blockingvalue to patient table, while also limiting the size of the blocking table.

In most situations, using the UUID as the primary key is probably fine, I was just worried about larger STLTs with millions of records and the performance of the queries.

Reply:
This makes sense! I think the justification is a good call. Thanks for explaining!